fabric-events-lifecycle-v0/src/main/java/net/fabricmc/fabric/api/event/client/HudRenderCallback.java

```diff
@@ -0,0 +1,22 @@
+package net.fabricmc.fabric.api.event.client;
+
+import net.fabricmc.fabric.api.event.Event;
+import net.fabricmc.fabric.api.event.EventFactory;
+
+public interface HudRenderCallback {
+	/** Called after rendering the status effects overlay when rendering the hud. */
+	public static final Event<HudRenderCallback> EVENT = EventFactory.createArrayBacked(HudRenderCallback.class,
+		listeners -> delta -> {
+			for (HudRenderCallback event : listeners) {
+				event.render(delta);
+			}
+		}
+	);
+
+	/**
+	 * Called when the hud is rendered, which is displayed in game, in a world.
+	 *
+	 * @param delta The number of frames passed since the last full tick
```

Comment:
This still does not look right, isn't the tick delta more like the interpolated progress between previous tick and current tick ?

Reply:
Well I think it is, but honestly, now I don't know how to phrase it.

Reply:
just call it partial (ticks)

Reply:
No, it is a tick delta

Reply:
*partial ticks* is not yarn terminology last I checked.

Reply:
Just leave it delta. Leaving a mention to [`MathHelper.lerp`](https://github.com/FabricMC/yarn/blob/b8403e7661122b2e81507c420dec904ddc46a8de/mappings/net/minecraft/util/math/MathHelper.mapping#L137) may be helpful.

Reply:
Edited as I don't really know how to describe `delta` anymore.